std_msgs/Header header
geometry_msgs/Vector3 body_rate
float32 thrust